rmgpy/rmg/main.py

```diff
@@ -804,6 +807,41 @@ def execute(self, **kwargs):
         
         self.finish()
     
+    def check_model(self):
+        """
+        Run checks on the RMG model
+        """
+        #check that no two species in core or edge are isomorphic
+        for i,spc in enumerate(self.reactionModel.core.species):
+            for j in xrange(i):
+                spc2 = self.reactionModel.core.species[j]
+                if spc.isIsomorphic(spc2):
+                    raise CoreError('Although the model has completed, species {0} is isomorphic to species {1} in the core.  Please open an issue on GitHub with the following output: {2}\n{3}'.format(spc.label,spc2.label,spc.toAdjacencyList(),spc2.toAdjacencyList()))
```

Comment:
Have you tested this? Is a newline character needed before `{2}`?

Reply:
You're correct that it doesn't put it the second adjacencyList on the same line the first ends on, but I think it looks nicer with the extra space between the lists.  I have however added one before the first one as otherwise it puts the text and the first line of the adjacencyList on the same line.